NetKAN/TweakScaleRescaled.netkan

```diff
@@ -0,0 +1,32 @@
+spec_version: v1.18
+identifier: TweakScaleRescaled
```

Comment:
If you name the DLL `TweakScaleRescaled.dll` instead of `Scale.dll`, then CKAN would be able to detect manual installs properly. Seems like this would be an OK time to do it since this fork is still relatively new. Or would something break?

Reply:
well shoot, resolve doesn't post the comment I just wrote.

This could break mods that have a hard dependency on the dll (e.g. the TweakScale partmodule) or NEEDS[Scale] - the latter is easy enough to work around.  But I'd rather just make CKAN's autodetection systems better globally.